src/components/singleElement/H3.ts

```diff
@@ -0,0 +1,16 @@
+import setAttributes from '../../utils/setAttributes';
+
+interface Props {
+  h3?: {
+    class?: string;
+  };
+  text: string;
+}
+
+export const H3 = ({ h3, text }: Props): HTMLElement => {
+  const $h3 = document.createElement('h2');
```

Comment:
오타겠죠?ㅎㅎ
```suggestion
  const $h3 = document.createElement('h3');
```
단순한 텍스트 단위도 컴포넌트로 분리해보려고 노력하신 점 정말 좋아요 👍 👍 👍 
그런데 h2,h3는 굳이 다른 컴포넌트로 분리될만큼 하나의 컴포넌트에 큰 역할이 있는 것 같진 않네용..?
만약 앱내의 h1이 사용이 추가되면 h1컴포넌트도 또 추가되어야하는 점이 불필요할 것 같구요.
따라서 컴포넌트의 재사용의 이점을 조금 더 활용해보면 어떨까요?
저라면 Text컴포넌트를 만든 후 prop 으로 어떤 tag로 쓰여질지 (h3 or h2 or h1...) prop 으로 받아서 조금 더 확장성 있는 컴포넌트를 만들어볼 것 같아요.

Reply:
아래에 있는 span, p 컴포넌트도 Text 컴포넌트 하나로 대체 가능해보여요~!

Reply:
(+ 태그 단위로 컴포넌트를 분리하시기 보다는 기능 단위로 한번 분류해봐주세요 :) )